godot-core/src/builtin/vectors/vector_macros.rs

```diff
@@ -517,3 +791,104 @@ macro_rules! impl_from_tuple_for_vector4x {
         }
     };
 }
+
+/// Implements functions present on floating-point 2D and 3D vectors.
+macro_rules! impl_vector2_vector3_fns {
+    (
+        // Name of the vector type.
+        $Vector:ty,
+        // Names of the components, with parentheses, for example `(x, y, z, w)`.
+        ($($comp:ident),*)
+    ) => {
+        impl $Vector {
+            /// Returns the angle to the given vector, in radians.
+            #[inline]
+            pub fn angle_to(self, to: Self) -> real {
+                self.to_glam().angle_between(to.to_glam())
+            }
```

Comment:
The existing code is already like this, but I wonder if we should use the same pattern as for quaternions? :thinking: 

https://github.com/godot-rust/gdext/blob/4f9f47f8ba6a9f0103bab62af5436f96244cbad8/godot-core/src/builtin/quaternion.rs#L58-L60

Reply:
The glam type isn't know so like this?
```rust
pub fn angle_to(self, to: Self) -> real {
    self.glam2(&to, |a, b| a.angle_between(b))
}
```

Reply:
Yep, that's great too!